.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+name: Close Stale Issues
```

Comment:
```suggestion
name: Update Stale Issues
```

Not sure if this name is shown anywhere but we've had people get touchy about stalebot in the past, and I would like to avoid that to the extent possible

Reply:
ah interesting, as in they felt the "stalebot closed your issue" language was harsh?

Reply:
I don't remember the details but yeah something like that. https://github.com/certbot/certbot/pull/7133 for example.